mock_storage_test.go

```diff
@@ -25,33 +25,35 @@ import (
 // MockStorage structure represents a mocked implementation of Storage
 // interface to be used by tested consumer
 type MockStorage struct {
-	databaseInitializationCalled    int
-	databaseCleanupCalled           int
-	databaseDropTablesCalled        int
-	databaseDropIndexesCalled       int
-	getLatestKafkaOffsetCalled      int
-	printNewReportsForCleanupCalled int
-	cleanupNewReportsCalled         int
-	printOldReportsForCleanupCalled int
-	cleanupOldReportsCalled         int
-	closeCalled                     int
-	writeReportCalled               int
+	databaseInitializationMigrationCalled int
+	databaseInitializationCalled          int
+	databaseCleanupCalled                 int
+	databaseDropTablesCalled              int
+	databaseDropIndexesCalled             int
+	getLatestKafkaOffsetCalled            int
+	printNewReportsForCleanupCalled       int
+	cleanupNewReportsCalled               int
+	printOldReportsForCleanupCalled       int
+	cleanupOldReportsCalled               int
+	closeCalled                           int
+	writeReportCalled                     int
 }
 
 // NewMockStorage constructs new mock storage instance
 func NewMockStorage() MockStorage {
 	return MockStorage{
-		databaseInitializationCalled:    0,
-		databaseCleanupCalled:           0,
-		databaseDropTablesCalled:        0,
-		databaseDropIndexesCalled:       0,
-		getLatestKafkaOffsetCalled:      0,
-		printNewReportsForCleanupCalled: 0,
-		cleanupNewReportsCalled:         0,
-		printOldReportsForCleanupCalled: 0,
-		cleanupOldReportsCalled:         0,
-		closeCalled:                     0,
-		writeReportCalled:               0,
+		databaseInitializationMigrationCalled: 0,
```

Comment:
maybe change to databaseInitMigrationCalled

Reply:
fixed, thank you